app/components/forms/wizard/basic-details-step.js

```diff
@@ -83,6 +83,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return this.data.event.state !== 'published';
   }),
 
+  ticketNumber: computed('data.event.tickets', function() {
+    return this.data.event.tickets.length > 0 ? true : false;
+  }),
+
+  unsavedEvent: computed('data.event.identifier', function() {
+    return !this.data.event.identifier; 
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6112249)